### ISSUE TITLE
Keyboard navigation is not working as expected when window is resized, column is grouped and grid is scrolled

### ISSUE THREAD

**Issue by kacheshmarova** [IgniteUI/igniteui-angular#4735]
## Description  
When  browser window is resized and grid is grouped and scrolled keyboard navigation is not working as expected.

 * igniteui-angular version: 
 * browser: N/A

## Steps to reproduce  

1. Run the attached sample in a browser window that is not maximized.
2. Scroll either vertically or horizontally to the middle.
3. Maximize browser window.
4. Group by column "3" .
5. Scroll vertically to row group 64-3. Click on any cell from row with id 64

## Result  
Cell is not getting focused and there is no keyboard navigation.

## Expected result  
Cell should be focused and keyboard navigation should be working as expected.

## Attachments  
![KeyboardNav](https://user-images.githubusercontent.com/13034836/57515479-6ce88600-731b-11e9-930e-5a11fd58d285.gif)
[igx-grid-keyboard.zip](https://github.com/IgniteUI/igniteui-angular/files/3165711/igx-grid-keyboard.zip)
